test/s3path.jl

```diff
@@ -432,6 +432,19 @@ function s3path_tests(config)
         rm(json_path)
     end
 
+    @testset "Arrow <-> S3Path (de)serialization" begin
+        ver = String('A':'Z') * String('0':'5')
+        paths = [
```

Comment:
Ensure that is `paths` is a `Vector{S3Path}` and not `Vector{S3Path{Nothing}}`
```suggestion
        paths = S3Path[
```

Reply:
Speedy